save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/UserSettingsOrganizationsMenuView.kt

```diff
@@ -1,22 +1,46 @@
 package com.saveourtool.save.frontend.components.views.usersettings
 
 import com.saveourtool.save.domain.Role
+import com.saveourtool.save.entities.OrganizationDto
 import com.saveourtool.save.frontend.components.basic.cardComponent
+import com.saveourtool.save.frontend.components.modal.displayConfirmationModal
+import com.saveourtool.save.frontend.components.modal.mediumTransparentModalStyle
+import com.saveourtool.save.frontend.externals.fontawesome.faTrashAlt
+import com.saveourtool.save.frontend.externals.fontawesome.fontAwesomeIcon
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.frontend.utils.noopLoadingHandler
 import com.saveourtool.save.v1
-import csstype.ClassName
 
+import csstype.ClassName
+import org.w3c.fetch.Response
 import react.FC
 import react.dom.html.ReactHTML.a
+import react.dom.html.ReactHTML.button
 import react.dom.html.ReactHTML.div
 import react.dom.html.ReactHTML.h1
 import react.dom.html.ReactHTML.img
 import react.dom.html.ReactHTML.li
 import react.dom.html.ReactHTML.ul
+import react.useState
+
+import kotlinx.coroutines.launch
 
-@Suppress("MISSING_KDOC_TOP_LEVEL", "TOO_LONG_FUNCTION")
+@Suppress("MISSING_KDOC_TOP_LEVEL", "TOO_LONG_FUNCTION", "LongMethod")
 class UserSettingsOrganizationsMenuView : UserSettingsView() {
     private val organizationListCard = cardComponent(isBordered = false, hasBg = true)
-    override fun renderMenu(): FC<UserSettingsProps> = FC { props ->
+
+    override fun renderMenu(): FC<UserSettingsProps> = FC { _ ->
+        val (deleteOrganization, setDeleteOrganization) = useState(OrganizationDto.empty)
+        val windowOpenness = useWindowOpenness()
+        displayConfirmationModal(
+            windowOpenness,
+            "Deletion Organization",
+            "Please confirm deletion of ${deleteOrganization.name}. " +
+                    "Note! This action deletes all the projects of this organization and the organization itself!",
+            mediumTransparentModalStyle
+        ) {
+            deleteOrganization(deleteOrganization)
```

Comment:
can we rename field `deleteOrganization` to `organizationToDelete` to avoid similar names